packages/api-clients/open-api/catalogApi.yml

```diff
@@ -1019,6 +1019,56 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  /eservices/{eServiceId}/update:
+    parameters:
+      - $ref: "#/components/parameters/CorrelationIdHeader"
+    post:
+      security:
+        - bearerAuth: []
+      tags:
+        - process
+      summary: Update an e-service description
+      operationId: updateEServiceDescription
```

Comment:
just a mild warning on future-proofness, without strong opinions.

I see why it has been created a specific event for the description update, it should not have major impacts and can be ignored by almost all consumers. 
But if (when) more details will need an update on a published eservice, handling different changes in the same endpoint could be a bit convoluted (it would require to understand which fields have been changed, or an endpoint for each change set).

Having said that, if you have already taken in considerations these points, ✅  for me

Reply:
> just a mild warning on future-proofness, without strong opinions.
> 
> I see why it has been created a specific event for the description update, it should not have major impacts and can be ignored by almost all consumers. But if (when) more details will need an update on a published eservice, handling different changes in the same endpoint could be a bit convoluted (it would require to understand which fields have been changed, or an endpoint for each change set).
> 
> Having said that, if you have already taken in considerations these points, ✅ for me

In that case, maybe it would be better to create a more generic `EServiceUpdated` event which updates the whole eservice? In this case it would only update the description but in the future could be used to update all its fields

Reply:
`EServiceUpdated` is too generic and it's difficult to use (like we had in event_version 1).

IMO it's ok to keep this event, and in case create specific events for fields that would have different consequences.
It's very likely that the endpoint will change, but it is not a big deal (until it is added to m2m apis)